jvb/src/main/java/org/jitsi/videobridge/util/ByteBufferPool.java

```diff
@@ -282,4 +281,14 @@ public static boolean statisticsEnabled()
         return enableStatistics;
     }
 
+    public static void enableBookkeeping(boolean enable)
+    {
+        bookkeepingEnabled = enable;
```

Comment:
I suggest that disabling bookkeeping should clear the `bookkeeping` and `returnedBookkeeping` maps